src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -244,6 +245,7 @@ void mutex_init() {
   def(RawMonitor_lock              , PaddedMutex  , nosafepoint-1);
 
   def(Metaspace_lock               , PaddedMutex  , nosafepoint-3);
+  def(MetaspaceCritical_lock       , PaddedMonitor, nosafepoint-1, true);
```

Comment:
You don't need the true parameter.  That's the default for nosafepoint locks.

Reply:
Oh, okay. Fixed.